app/services/usps_in_person_proofing/mock.rb

```diff
@@ -1,26 +1,36 @@
+require './spec/support/usps_ipp_fixtures'
```

Comment:
It catches my attention that we're loading spec code from the app code, vs. the other way around. I suppose it's probably not an issue? The alternative could be to move the mock data somewhere in this folder (e.g. `app/services/usps_in_person_proofing/mock/responses/request_facilities.json`), but I also don't see much prior art for that, so could leave it as-is.

Reply:
(Only after I wrote this did I notice you already self-reported at https://github.com/18F/identity-idp/pull/6629#discussion_r943882063 😄 )

Reply:
I like the idea of having a file in `mock/` for this, it does feel weird to be pulling in a spec fixture.

Reply:
Okay, I moved things around in baff25e66

1. Move all of the response json files to app/services/usps_in_person_proofing/mock/responses/
2. Move the UspsIppFixtures class to UspsInPersonProofing::Mock::Fixtures
3. Update all of the references to the fixtures to the new class